### ISSUE TITLE
--versionStyle Not Working

### ISSUE THREAD

**Issue by debasisdev** [DanielCech/VersionIcon#1]
@DanielCech Hello Daniel, I stumbled upon your kool project today and wanted it to integrate to my App. Unfortunately, I wanted to show only Build Number on my icon - Hence I tried something like this in my Build Phase Script:

```
"${PODS_ROOT}/VersionIcon/Bin/VersionIcon" --appIconOriginal AppIcon --ribbon Blue-TopRight.png --title Devel-TopRight.png --titleSize 0.15 --fillColor 66ffff --strokeWidth 0.005 --verticalTitlePosition 0.1 --versionStyle buildOnly
```

It resulted in an error prompting: `💥 error: invalid argument: Invalid versionStyle argument`

Without `--versionStyle buildOnly` the command works flawlessly in a jiffy. Could you please double check, what's the issue?

**Comment by debasisdev** [comment]
Just FYI: `--versionStyle` only works with `dash` - all other options results in an error 😢

**Comment by DanielCech** [comment]
Hey Debasis, thanks for your message. I am glad you are using VersionIcon. I’ll check it and let you know :-).
Cheers,

Dan

**Comment by DanielCech** [comment]
I released a new version 1.0.8 and this bug is fixed there. Thank you for reporting a bug.
Please use `pod install --repo-update`
